### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by aviflax**
Biggest bug fixed herein is that the snap-to-virtual-grid feature wasn’t working at all. But I found a few other bugs while looking into that.

It’d probably be best/easiest to review this commit-by-commit.

### CODE REVIEW

File: src/main/fc4c/integrations/structurizr/express/edit.clj [hunk 1]
```diff
@@ -184,19 +212,28 @@
 (defn snap-coords
   "Accepts a seq of X and Y numbers, and config values and returns a string in
   the form \"x,y\"."
+  ; TODO: it’s inconsistent that the min-margin is passed in as an arg but the
```
Comment (sgerrand) [inline src/main/fc4c/integrations/structurizr/express/edit.clj]: 🙋 Please migrate this comment to an issue instead.

File: src/main/fc4c/integrations/structurizr/express/spec.clj [hunk 2]
```diff
@@ -9,7 +9,10 @@
 
 (def ^:private aliases-to-namespaces
   {'st 'structurizr
+   'sy 'structurizr.system
+   'sp 'structurizr.person
    'sc 'structurizr.container
+   'sm 'structurizr.component
```
Comment (sgerrand) [inline src/main/fc4c/integrations/structurizr/express/spec.clj]: 🙋 Suggest sorting this list.

File: src/main/fc4c/integrations/structurizr/express/edit.clj (hunk 1 above)
Comment (aviflax) [reply src/main/fc4c/integrations/structurizr/express/edit.clj]: Good idea, will do — thanks!

**Comment by aviflax** [comment]
> I strongly suggest moving the inline "TODO"s scattered through the codebase to issues so that others can help contribute to this project.

Good idea, I’ll do so in a follow-up PR. Thanks!

**Comment by aviflax** [comment]
I’ve decided to open a follow-up PR with those commits that will raise test coverage; the gains are fairly small but the changes are non-trivial.

**Pull request merged**